### DISCUSSION TITLE
Rest Api Food data

### DISCUSSION THREAD

**Discussion by Marfio** [mealie-recipes/mealie#2032] [Q&A] [answered]
I think is a great application for manage my recipes! I want to congratulate the author. 

I was searching in API reference the way to GET the Food Data. 

Is there any way to get de food data directly? 

Thanks

**Comment by RealFoxie** [comment, marked as answer]
You can see your own api docs with `/docs`
For food you have the following endpoints:
![afbeelding](https://user-images.githubusercontent.com/52831920/212469710-8c81147e-d961-43cc-9500-dbbdc17ab593.png)

Is that what you mean? (this is for 1.0, not sure about 0.5)

**Reply by Marfio** [reply]
Yes! 
I was searching on iPad and only showed me Maelia (v0.5.1) 
Now on my computer I can see Maelia (v1.0.0beta5)

Thank you very much!
